<!DOCTYPE html>
<html lang="en">

<head>
    <!-- Required meta tags always come first -->
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <!-- Bootstrap CSS -->
	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
	<link href="//maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css" rel="stylesheet" />
	<link rel="stylesheet" href="css/css.css">
    <title>Alise Ponsero</title>
</head>

<body id="page-top" data-spy="scroll" data-target=".navbar" data-offset="-50">
<nav id="nav" class="navbar fixed-top navbar-expand-sm">
    <div class="container">
        <a href="#page-top" class="navbar-brand">Alise PONSERO</a>
        <a class="navbar-toggler" data-toggle="collapse" data-target=".navbar-collapse">☰</a>
        <div class="navbar-collapse collapse">
            <ul class="ml-auto navbar-nav">
                <li class="nav-item"><a href="#About" class="nav-link">About</a></li>
				<li class="nav-item"><a href="#Activities" class="nav-link">Portfolio</a></li>
				<li class="nav-item"><a href="#Contact" class="nav-link">Contact</a></li>
            </ul>
        </div>
    </div>
</nav>
<!-- /.navbar -->

<!-- page content -->
<section class="first">
	<div class="container h-100">
		<div class="row h-100 align-items-center">
			<div class="col-12 text-center">
				<h1 class="font-weight-light">Professional portfolio</h1>
				<a href="#About" class="btn btn-outline-dark btn-lg" role="button" aria-pressed="true">Learn more</a>
			</div>
		</div>
	</div>
</section>

<section id="About">
<div class="container" id="container-main">
<div class="col-12 text-center">
  <h1 class="font-weight-light">About me</h1>
  	<div>
		<hr>
	</div>
</div>
<div class="d-none d-md-block">
	<div class="d-flex justify-content-around flex-wrap">
		<img src="ressources/logo_sample_BW_small.png" class="img-fluid" alt="image2">
		<img src="ressources/Logo_integration_BW_small.png" class="img-fluid" alt="image3">
		<img src="ressources/logo_omics_BW_small.png" class="img-fluid" alt="image4">
	</div>
</div>
 <div class="col-12 text-center">
		<p> </br></br> I am an bioinformatician broadly interested in environmental microbiology and metagenomics.</br></br>
		My research experience encompass different topics and interests. I realized a master's degree in fundamental and applied microbiology at the university of Rennes 1, France. 
		During this master's degree, I realized an intership placement at the university of Tennesse, USA in the laboratory of Dr. S.W. Wilhelm, working on phage-host interactions
		between cyanobacteria and cyanophages.</br></br>
		I went back to France to realize a PhD in molecular and cellular biology in the CEA (center for atomic energy) of Saclay, France. My dissertation focused
		on understanding intracellular fluxes of glutathion and its impact on protein secretion in yeast.</br></br>
		Convinced of the importance of computer science and data analysis skills in today's biological research, I decided to complete a second master's degree in computer science, at the
		university of Rennes 1, France. During this time, I add the opportunity to perform a placement at the University of Arizona, in the laboratory of Dr. Bonnie Hurwitz.
		At the end of my master's degree, I joined her lab as a postdoctoral fellow, working on several projects and tools for the analysis of metagenomics datasets.
	</div>
</div>

</section>
<section id="Activities">
	<div class="container" id="container-main">
			<div class="col-12 text-center">
				<h1 class="font-weight-light">My activities</h1>
				<div>
					<hr>
				</div>
			</div>

              <div class="row">
                <div class="col-xs-12 ">
                  <nav>
                    <div class="nav nav-tabs nav-fill" id="nav-tab" role="tablist">
                      <a class="nav-item nav-link active" id="nav-current-tab" data-toggle="tab" href="#nav-current" role="tab" aria-controls="nav-current" aria-selected="true">Current works</a>
                      <a class="nav-item nav-link" id="nav-past-tab" data-toggle="tab" href="#nav-past" role="tab" aria-controls="nav-past" aria-selected="false">Past research</a>
                      <a class="nav-item nav-link" id="nav-other-tab" data-toggle="tab" href="#nav-other" role="tab" aria-controls="nav-other" aria-selected="false">Other works</a>
                    </div>
                  </nav>
                  <div class="tab-content py-3 px-3 px-sm-0" id="nav-tabContent">
                  
				  <div class="tab-pane fade show active" id="nav-current" role="tabpanel" aria-labelledby="nav-current-tab">
					<!-- Current scientific interests -->
						<div class="col-12 text-center">
						  <h3 class="font-weight-light">Current research interests</h3>
						</div>
						<div class="col-12 text-left">
							<h5 class="font-weight-light" >Viral hunting in metagenomes</h5>
							<p></br>Understanding the role and distribution of viruses in ecosystems is critical and metagenomics offers new avenues. 
							To explore the viral biodiversity and ecology, a number of bioinformatic tools perform a high-level taxonomic (viral or cellular origin) assignment of metagenomic sequences. 
							They aim to provide means to collect all viral sequences in a metagenome and help the discovery of new viral groups. 
							While many tools rely on references databases, new approaches using machine learning and pattern detection techniques have been suggested. 
							The idea is machine learning model to learn to identify a set of features that signal a viral origin to generalize the identification of all viral sequences.
							As users of these tools, we aim to explore the promise and current limitations of viral detection in metagenomic datasets.
							</p>

							<h5 class="font-weight-light" >kmer-based de novo metagenomic comparisons</h5>
							<p></br>Over the last decade, scientists have generated petabytes of genomic data to uncover the role of microbes in dynamic living systems. 
							Yet, to understand the underlying biological principles that guide the distribution of microbial communities, massive ‘omics datasets need to be compared with 
							environmental factors to find linkages across space and time. 
							In order to examine the complete genomic content, metagenomic samples can be compared using their sequence signature (or frequency of k-mers). 
							This approach relies on three core tenets of k-mer–based analytics: (i) closely related organisms share k-mer profiles and cluster together, making taxonomic 
							assignment unnecessary; (ii) k-mer frequency is correlated with the abundance of an organism; and (iii) k-mers of sufficient length can be used to distinguish 
							specific organisms.
							</p>

							<h5 class="font-weight-light" >FAIR principle applied to metagenomic datasets</h5>
							<p></br>Oceanographic research cruises provide abundant physical, chemical, and biological data, using a wide range of methods and equipment, 
							very often through the collaborative efforts of scientists from a range of disparate disciplines. These research endeavors span across a broad 
							array of disciplines and are critical to investigating questions related to the interplay between biological, geological, and chemical processes in the 
							ocean systems over space and time. The advent of sequencing technologies allows for an analysis of gene expression in a variety of environmental settings, 
							to measure the distribution and significance of metabolites and lipids in organisms and the environment.
								Importantly, although scientists carefully curate and share their data with collaborators to advance individual studies and publications, no systematic, 
								unifying framework currently exists to integrate ‘omics data with physical, geological, geochemical, and biological datasets standardly used by the broader 
								geoscience community. In this context, the development of resources to promote the publication of oceanic ‘omics datasets along with their physio-chemical 
								information is critical.

								Moreover, the Planet Microbe project aims to provide community tools to support the publication and analysis of new oceanic datasets. 
								Modern data publication requires the user to meet FAIR data standards to ensure data are Findable Accessible, Interoperable, and Reusable (FAIR), however, 
								few tools are currently available for the user to ensure the data are meeting these important standards. Planet Microbe aims to provide tools to evaluate private 
								datasets against these standards and promote the integration of new data in standard data repositories.
							</p>
						</div>
					</div>

                    <div class="tab-pane fade" id="nav-past" role="tabpanel" aria-labelledby="nav-past-tab">
                      <!-- past scientific interests -->
						<div class="col-12 text-center">
						  <h3 class="font-weight-light">Past research interests</h3>
						</div>
						<div class="col-12 text-left">
							<h5 class="font-weight-light" >redox biology and protein secretion in the yeast S. cerevisiae</h5>
							<p></br>The endoplasmic reticulum (ER) is the first intracellular compartment of the protein secretion pathway. 
							Protein maturation in this compartment involves protein folding and post-traductionnal modification including formation of disulfide bonds. 
							The formation of disulfide bonds is operated by a highly conserved redox relay made of the thiol oxidase Ero1 and the protein disulfide isomerase (PDI). 
							Ero1p catalyzes disulfide bond formation and relays them by thiol-disulfide exchange to PDI, which in turn oxidizes substrates. 
							Isomerization and terminal reduction of non-native disulfide bonds both rely on a reduction system that remains to be formally identified. 
							Studies however suggest the importance of reduced glutathione in this reducing system. GSH is small redox tripeptide exclusively synthesized in the cytosol. 
							In this study we (i) describe the main parameters of glutathione traffic across the ER membrane (ii) identify the main actors involved in the transport and (iii) 
							analyze the physiological impact of a modification of the ER glutathione redox state
							</p>

							<h5 class="font-weight-light" >marine viral ecology</h5>
							<p></br>A 6-month internship at the university of Tennessee gave me the opportunity to work on the influence of water temperature on the population dynamics of cyanophages. 
							This placement was a great opportunity for me to join a marine scientific expedition to collect samples (POWOW cruise 1).
							</p>

							<h5 class="font-weight-light" >microbe-plant interaction in soil</h5>
							<p></br>Two short-term internships allowed me to investigate the interactions of bacteria or fungi and plants, 
							first in the study of the molecular mecanisms allowing the  establishement of the actinomycete symbiosis with the common alder. 
							A second short research project was performed on the competition of pseudomonas aeruginosa and the fungi responsible for the take-all disease of wheat.
							</p>
						</div>
					</div>

                    <div class="tab-pane fade" id="nav-other" role="tabpanel" aria-labelledby="nav-other-tab">
                     
					<!-- Other interests -->
						<div class="col-12 text-center">
						  <h3 class="font-weight-light">Other works</h3>
						</div>
						<div class="col-12 text-left">
							<h5 class="font-weight-light" >Scientific heritage</h5>
							<p></br>The endoplasmic reticulum (ER) is the first intracellular compartment of the protein secretion pathway. 
							Protein maturation in this compartment involves protein folding and post-traductionnal modification including formation of disulfide bonds. 
							The formation of disulfide bonds is operated by a highly conserved redox relay made of the thiol oxidase Ero1 and the protein disulfide isomerase (PDI). 
							Ero1p catalyzes disulfide bond formation and relays them by thiol-disulfide exchange to PDI, which in turn oxidizes substrates. 
							Isomerization and terminal reduction of non-native disulfide bonds both rely on a reduction system that remains to be formally identified. 
							Studies however suggest the importance of reduced glutathione in this reducing system. GSH is small redox tripeptide exclusively synthesized in the cytosol. 
							In this study we (i) describe the main parameters of glutathione traffic across the ER membrane (ii) identify the main actors involved in the transport and (iii) 
							analyze the physiological impact of a modification of the ER glutathione redox state
							</p>

							<h5 class="font-weight-light" >Scientific outreach and communication</h5>
							<p></br>
							</p>
						</div>
					</div>
					  
					</div>
                  </div>
                
                </div>
              </div>
        </div>
      </div>

</section>


<section id="Contact">
	<div class="container" id="container-main">
		<div class="row h-100 align-items-center">
			<div class="col-12 text-center">
				<h1 class="font-weight-light">Contact me</h1>
				<div>
					<hr>
				</div>
			</div>

			<div class="col-12 text-center">
				<div class="d-flex justify-content-around flex-wrap">
					<!-- Twitter -->
					<span class="fa-stack fa-lg">
					  <a href="https://twitter.com/APonsero" style="color:black"aria-hidden="true">
					  <i class="fa fa-square-o fa-stack-2x"></i>
					  <i class="fa fa-twitter fa-stack-1x"></i></a>
					</span>
					<!-- Mail -->
					<span class="fa-stack fa-lg">
					  <a href="mailto:aponsero@email.arizona.edu" style="color:black"aria-hidden="true">
					  <i class="fa fa-envelope fa-stack-1x"></i></a>
					</span>
					<!-- LinkedIn -->
					<span class="fa-stack fa-lg">
					  <a href="https://www.linkedin.com/in/alise-ponsero-843b953b/" style="color:black"aria-hidden="true">
					  <i class="fa fa-square-o fa-stack-2x"></i>
					  <i class="fa fa-linkedin fa-stack-1x"></i></a>
					</span>
					<!-- github -->
					<span class="fa-stack fa-lg">
					  <a href="https://github.com/aponsero" style="color:black"aria-hidden="true">
					  <i class="fa fa-square-o fa-stack-2x"></i>
					  <i class="fa fa-github fa-stack-1x"></i></a>
					</span>
				</div>
		</div>	
	</div>
</section>

<section class="last">
	<div class="container h-10">
		<div class="row h-100 align-items-center">
		</div>
	</div>
</section>
<!-- Footer -->
<footer class="page-footer font-small bg-dark">
  <!-- Footer Elements -->
  <div class="container">
    <!--Grid row-->
    <div class="row">
		<div class="col-12 text-center text-light">
			<p>This website was designed and hand-made by Alise Ponsero. <a href="https://github.com/aponsero/Web-page-perso">Fork the code in github !</a></p> 
		<div>
    </div>

  <!-- Copyright -->
  <div class="footer-copyright text-center text-light">Copyright © Alise Ponsero. All Rights Reserved
  </div>
  <!-- Copyright -->

</footer>
<!-- Footer -->

<!-- /page content -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.13.0/umd/popper.min.js"></script>
  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js"></script>
  <script src="https://rawgit.com/bassjobsen/affix/master/assets/js/affix.js"></script>
  <script>
  /* affix the navbar after scroll below header */
	$('nav').affix({
		  offset: {
			top: $('#About').offset().top
		  }
	});	
  </script>

</body>
</html>